templates/skeleton/app/routes/account/login.tsx

```diff
@@ -74,3 +66,25 @@ export default function Login() {
     </Form>
   );
 }
+
+// Remix v1 implementation of ErrorBoundary; for Remix v2, this will need to be updated according to the following documentation:
+// https://remix.run/docs/en/1.15.0/pages/v2#catchboundary-and-errorboundary
```

Comment:
This is a case where the JSDoc syntax makes sense, I think:

```ts
/**
 * Remix v1 implementation of ErrorBoundary. This will be updated to the v2 
 * ErrorBoundary component in a future release.
 * @see https://remix.run/docs/en/1.15.0/pages/v2#catchboundary-and-errorboundary
 */
```

Reply:
An interesting side effect of multi-line comments is that TypeScript ends up taking those comments and using them as the TS definition for the function. So, for better or worse, if you were to hover over that function, you would get that text in an Intellisense popup. 

Not sure if that's bad, good, or neutral, though. Thoughts?

Reply:
+1 good :)